pkg/cmd/util/clientcmd/errors.go

```diff
@@ -0,0 +1,50 @@
+package clientcmd
+
+import "strings"
+
+const (
+	unknownReason                     = 0
+	noServerFoundReason               = 1
+	certificateAuthorityUnknownReason = 2
+
+	certificateAuthorityUnknownMsg = "The server uses a certificate signed by unknown authority. You may need to use the --certificate-authority flag to provide the path to a certificate file for the certificate authority, or --insecure-skip-tls-verify to bypass the certificate check and use insecure connections."
+	notConfiguredMsg               = `OpenShift is not configured. You need to run the login command in order to create a default config for your server and credentials:
+  osc login
+You can also run this command again providing the path to a config file directly, either through the --config flag of the OPENSHIFTCONFIG environment variable.
+`
+)
+
+func GetPrettyMessageFor(err error) string {
+	reason := detectReason(err)
+
+	switch reason {
+	case noServerFoundReason:
+		return notConfiguredMsg
+
+	case certificateAuthorityUnknownReason:
+		return certificateAuthorityUnknownMsg
+	}
+
+	return err.Error()
```

Comment:
Be sure to handle the case where `err` is nil.  I think this just tries to call `Error` on it which would probably result in a panic for most implementations.